src/Timer.js

```diff
@@ -23,19 +23,51 @@ Button.defaultProps = {
   bg: '#333'
 }
 
+const direction = {
+  normal: 0,
+  countdown: 1
+}
+
+const DEFAULT_COUNTDOWN = 1800
+
 class Timer extends React.Component {
   state = {
     on: false,
     time: new Date().toLocaleTimeString(),
-    seconds: 0
+    seconds: 0,
+    directionMode: direction.normal,
+    countdownTime: DEFAULT_COUNTDOWN
```

Comment:
If this was initially set from props, would allow the mdx file to pass a value in from the main app

Reply:
Exactly, so we if have access to this value from mdx, we can replace this easily.

Reply:
So f.e. inside the mdx we would export a `time` or `presentationTime` value? And we can access it via `withDeck` for example? or would we define some sort of `settings` object similar to `theme`

Reply:
I think e.g. `export const timer = 30` would be nice to have in the mdx file, doesn't need to be a full settings object yet